src/main/java/htsjdk/samtools/CRAMContainerStreamWriter.java

```diff
@@ -15,6 +15,7 @@
 import htsjdk.samtools.cram.structure.ContainerIO;
 import htsjdk.samtools.cram.structure.CramCompressionRecord;
 import htsjdk.samtools.cram.structure.Slice;
+import htsjdk.samtools.cram.structure.slice.SliceType;
```

Comment:
Someday the `slice` package eventually contain other stuff ? Otherwise it seems strange to have `SliceType` but not `Slice` in there...

Reply:
yes - I hope to use that for the overall Slice refactor.  I can put elsewhere (structure?) if you prefer that I wait until then.

Reply:
If this is the right place its fine - just wanted to check.